lib/dolly/request.rb

```diff
@@ -101,6 +102,16 @@ def full_path resource
     def attachment_path resource, attachment_name
       "#{full_path(resource)}/#{attachment_name}"
     end
+
+    def log_request resource, response_code
+      log_value = ->(resource, response_code) { "Query: #{resource}, Response Code: #{response_code}" }
+      case response_code
+      when 200..399
+        Dolly.logger.info log_value[resource, response_code]
+      when 400..600
+        Dolly.logger.warn log_value[resource, response_code]
```

Comment:
not sure if this should be `error` as any of these will be a fatal error and stop the execution of any script